src/components/index.tsx

```diff
@@ -1,4 +1,5 @@
 export { NavBar } from './NavBar';
+export { default as EventCard } from './EventCard';
```

Comment:
can we align all exports to be the same as the others please? so you don't need export default in the EventCard itself

Reply:
Updated the export as the others.